app/mailers/game_mailer.rb

```diff
@@ -0,0 +1,5 @@
+class GameMailer < ApplicationMailer
+    def game_player_joined
+        # A player joined your game
+    end
```

Comment:
I'm a whitespace nazi and I see that this was indented with 4 spaces instead of 2.